bundles/org.openhab.binding.argoclima/.gitignore

```diff
@@ -0,0 +1,2 @@
+/.apt_generated/
```

Comment:
Don't think the .gitignore file should be included as it contains only specific files from your environment.

Reply:
Lol :) And here was me thinking it was the bootstrap script generating this ignorefile for me :) Will delete, thx!